src/lib.rs

```diff
@@ -1,564 +1,249 @@
-/*
-Firebase REST API
-*/
```

Comment:
This top-level crate documentation was a great idea. Maybe not in this PR, but eventually I highly recommend writing doc comments for everything, including top-level module documentation.